Osiris/Hooks.cpp

```diff
@@ -491,7 +491,9 @@ static bool __STDCALL dispatchUserMessage(LINUX_ARGS(void* thisptr, ) int messag
 {
     if (messageType == 7) // CS_UM_TextMsg
         InventoryChanger::onUserTextMsg(data, size);
-
+    else if (messageType == 46 || messageType == 47 || messageType == 48)
```

Comment:
You can use `UserMessageType` enum for that (4b88f690651ce2ffec23a4b6a5c767e32a4b0a96).